docs/course_materials/compound_data_types.html

```diff
@@ -47,23 +49,22 @@ <h1><b>
                     <h3><b>Basics</b></h3>
                     <ul>
                         <li>An ordered collection of values</li>
+                        <li>Index count starts from 0</li>
                         <li>Create an empty list</li>
                     </ul>
-                    <pre><code data-trim data-noescape class="hljs">
->>> l = list()
->>> type(l)
-list
->>> l = []
->>> type(l)
-list
+                    <pre><code data-trim data-noescape class="hljs lang-python">
```

Comment:
lang-python
didn't know this one exists :)

Reply:
I've found it in the highlight.js docs :)
We could also just use `python` or `language-python`. I've chose the middle one for readability.